app/models/call.rb

```diff
@@ -1,4 +1,5 @@
-class Call
+ class Call<Auditable::Base
```

Comment:
I think you'll want to change `class Call < Auditable::Base` to `class Call < ActiveRecord::Base` here.